internal/impl/kafka/enterprise/kafka_migrator_output.go

```diff
@@ -0,0 +1,499 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Licensed as a Redpanda Enterprise file under the Redpanda Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+// https://github.com/redpanda-data/connect/blob/main/licenses/rcl.md
+
+package enterprise
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/twmb/franz-go/pkg/kgo"
+	"github.com/twmb/franz-go/pkg/sasl"
+
+	"github.com/redpanda-data/benthos/v4/public/service"
+	"github.com/redpanda-data/connect/v4/internal/impl/kafka"
+)
+
+const (
+	rproDefaultLabel = "kafka_migrator_output"
+)
+
+func kafkaMigratorOutputConfig() *service.ConfigSpec {
+	return service.NewConfigSpec().
+		Beta().
+		Categories("Services").
+		Version("4.33.1").
+		Summary("A Kafka output using the https://github.com/twmb/franz-go[Franz Kafka client library^].").
```

Comment:
Should there be something more here to distinguish between the regular kafka_franz output and this one?

Reply:
I replaced `Kafka` with `Kafka Migrator`. Hope that's a good enough summary.